### ISSUE TITLE
train_from_dataset函数返回值为None

### ISSUE THREAD

**Issue by gentelyang** [PaddlePaddle/Paddle#19482]
train_from_dataset函数返回值为None，无法获取返回的cost等值，通过global_scope获取不方便，
希望能改一下这个函数能方便获取返回值。
  loss = exe.train_from_dataset(
                program=fleet.main_program,
                dataset=dataset,
                fetch_list=[self.avg_cost],
                fetch_info=["cost"],
                print_period=100,
                debug=False)

**Comment by wangchaochaohu** [comment]
我记录一下  后续会反馈改进 谢谢
